modules/ROOT/pages/api-development-studio.adoc

```diff
@@ -3,58 +3,74 @@ ifndef::env-site,env-github[]
+Import a RAML API specification either xref:import-api-specification-local-file.adoc[from your hard drive] or xref:import-api-specification-design-center.adoc[directly from Design Center] into the `/src/main/api/` directory in the structure of your project. This process creates a local copy of the API in your project's structure. The imported API works as a standalone attachment to your project, meaning that every additional upgrade requires a manual import.
 
-Whenever the API that you import as a local copy into your project's structure is updated outside your Studio environment, you must reimport the API into your project and regenerate all the flows based on the updated specification, risking creating duplicated copies of your specification.
+When the API imported as a local copy into your project's structure is updated outside your Studio environment, reimport the API into your project. Studio regenerates all the flows based on the updated specification. 
 
-You can only keep RAML API specifications as local copies within your project's structure.
+[NOTE]
+Duplicated copies of your specifications can be created on your local drive. You can only keep RAML API specifications as local copies within your project's structure.
 
 == API Specifications as API Specification Projects in Studio
 
-You can start xref:create-api-specification-studio.adoc[an API specification project in Studio and sync it with Design Center]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and Design Center.
+Start xref:create-api-specification-studio.adoc[an API specification project in Studio and sync it with Design Center]. Studio uses version control system (VCS) capabilities to keep your API specifications synchronized between your Studio workspace and Design Center.
 
-You can also import an API specification from Design Center as an API specification project with VCS-enabled capabilities to edit your API specification using Studio and sync your changes with Design Center:
+Import an API specification from Design Center as an API specification project with VCS-enabled capabilities to edit your API specification using Studio and sync your changes with Design Center:
 
 . Design an API specification in Design Center. +
-You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Code Editor].
+You can either xref:design-center::design-create-publish-api-visual-editor.adoc[create and publish an API specification with the Visual Editor] or xref:design-center::design-create-publish-api-raml-editor.adoc[with the Text Editor].
```

Comment:
See previous comment about this link text.

Reply:
See my previous comment about hard-coded in-line text.

Reply:
I meant specifically the suggestion I had about making just the "xxx Editor" the link text and making the links parallel.

https://github.com/mulesoft/docs-studio/pull/529#discussion_r1646770903

Reply:
OK! I got it now.
Thanks!